### PR TITLE
Sprint 5: Edit College

### PR DESCRIPTION
**Pull request by chroline**
**What changed:**
- added `/edit-college/:id` route

**How to test:**
- visit the route, with the ID of the college you want to test

CleanShot 2024-02-08 at 01 33 10

### CODE REVIEW

**Comment by chroline** [comment]
note: plz don't merge until #41 is merged :)

**Review by oliviasl — APPROVED** [review]
Nice!

**Pull request merged**
